en/installconfig-upgrade-to-2.md

```diff
@@ -17,11 +17,50 @@ Ubuntu to 16.04 LTS. In so doing, the MAAS database and MAAS configuration will
 be migrated. See the [Xenial Release Notes][xenial-release-notes-upgrading] for
 upgrading to Xenial.
 
-Prior to the upgrade all packages should be updated (`sudo apt-get update &&
-sudo apt-get dist-upgrade`). It is recommended that backups be made and that 
-a test environment (that mirrors your production environment) be upgraded first, to
-pre-empt any issues.
+Prior to the upgrade all packages should be updated
+(`sudo apt-get update && sudo apt-get dist-upgrade`). It is recommended that
+backups be made and that a test environment (that mirrors your production
+environment) be upgraded first, to pre-empt any issues.
 
+## IP range changes
+
+The concepts and terminology used to define IP ranges within MAAS change
+significantly between 1.9 and 2.x releases of MAAS. These changes are described
+below.
+
+MAAS 1.9 allows users to configure their managed networks to use either
+[static][1-9-static] or dynamic ranges. A static range is used by MAAS to allocate
+automatic (static) IP addresses while a dynamic range is used solely for
+DHCP.
+
+Static ranges are deprecated in MAAS 2.x. Instead, MAAS assumes full control
+of the address space for a network it manages with two exceptions:
+
+- An optionally defined [reserved range][ip-range] has been defined.
+- [Managed allocation][managed-subnet] in MAAS 2.3+ is disabled. 
```

Comment:
When...

Reply:
Not sure what happened to my whens, thanks!